lib/types.go

```diff
@@ -77,24 +77,9 @@ type NotifyResult struct {
 }
 
 // SignRequestResult is a JSON returned from transaction complete function (used in exposed method)
+// nolint: megacheck
```

Comment:
What is the error here making `nolint` required?

Reply:
Removing `SignRequestResult` causes this lint check to trigger
```
lib/library.go:261:9: undeclared name: SignRequestResult (typecheck)
```
Keeping `SignRequestResult` without the `// nolint` comment causes the following check to trigger
```
lib/types.go:80:6: type SignRequestResult is unused (megacheck)
```
Don't know what is going on with this. Will update once I do.